.github/workflows/labels.yml

```diff
@@ -0,0 +1,24 @@
+name: labels
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - .github/labels.yml
+      - .github/workflows/labels.yml
+  pull_request: # dry run only
+    paths:
+      - .github/labels.yml
+      - .github/workflows/labels.yml
+
+jobs:
+  labeler:
+    permissions:
+      contents: read
+      issues: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: crazy-max/ghaction-github-labeler@v5
```

Comment:
I will play the devils advocate here. I'm not sure if using an external action justifies the advantage (at least in daily usage). Maybe I'm not seeing the exact advantage here, but it seems to me that this can just be a 5 min operation if we ever wanted to fully rename all labels (through https://github.com/ava-labs/coreth/labels). Not sure if it's worth to run a 3rd party action in every push/pull. even though the action can be something secure it still adds to the CI overhead.

Reply:
The aim is to have source-controlled, reviewable (through PRs) Github labels. The rename part is only useful once, for example to rename default label names.

The action also runs in its own workflow only when labels yml files are changed, so it won't pollute on every commit.